core/templates/dev/head/css/oppia.css

```diff
@@ -2851,7 +2851,7 @@ div#ng-curtain {
 .nav > li > div.oppia-navbar-tab {
   font-family: "Capriola", "Roboto", Arial, sans-serif;
   font-size: 14px;
-  padding: 17px 12px 10px 12px;
+  padding: 17px 12px 10px 12px !important;
```

Comment:
We don't use !important in the code; that is essentially an override and makes it hard to keep track of the usual CSS priority order. Please try to find another way. Having an inline style tag is less problematic than using !important, so if that's what's needed to avoid !important then it's fine.

Reply:
P.S see https://github.com/oppia/oppia/wiki/Coding-style-guide#css

Reply:
HI @seanlip . Padding values is being taken from the third_party.css file and it seems to me that ,It is being ignored by Git so making any changes in that file is not getting tracked .So it would be fine,If I would add inline css to the elements.?

Reply:
You should not be making changes in third_party, yes. I think if the third-party file is overwriting your CSS, you can either add inline CSS to the elements or make this CSS selector more specific so that it gets a higher priority.

Reply:
@seanlip ,K Thanks 4 the help.